containers/eap/src/main/java/org/jboss/qa/jcontainer/eap/EapConfiguration.java

```diff
@@ -39,10 +39,7 @@ public Builder() {
 			super();
 			xms("1303m");
 			xmx("1303m");
-			//-XX:MaxPermSize has been removed from jdk17
```

Comment:
why did you revert this change

Reply:
moved to common code https://github.com/jboss-qa/jcontainer-manager/blob/jdk17-move-to-java-config/core/src/main/java/org/jboss/qa/jcontainer/JavaConfiguration.java#L118-L138